main.js

```diff
@@ -0,0 +1,6 @@
+let message ="hi this is my homework" 
+let x = ( message.length +100 - (message.length % 100) )
```

Comment:
Spacing and semicolon
```suggestion
let x = (message.length + 100 - (message.length % 100));
```

Reply:
პირდაპირ გამოთვალო არ გირჩევნია?
`message.length / 100` ეს ხომ მოგცემს რამდენი ცენტი უნდა გადაიხადო პირდაპირ?
- მაგალითად თუ მესიჯის ზომა 200 -ია ფასი 2 ცენტი იქნება
- მაგალითად თუ მესიჯის ზომა 156 -ია ფასი 1.56 ცენტია და უნდა იყოს 2 ცენტი, `Math.ceil` -ით შეგიძლია ზემოთ დაამრგვალო წილადი რიცხვი, უახლოეს მთელ რიცხვამდე